CMakeLists.txt

```diff
@@ -141,6 +142,10 @@ set(CMD_MAKE make)
 
 include(ExternalProject)
 
+if(USE_JEMALLOC)
```

Comment:
Seeing the comment about jemalloc and ASAN above, shouldn't there be a check done here that fails the configuration if both are enabled ? Otherwise we will likely have users that will get problematic builds.

Reply:
I agree, i'll just add a warning!

Reply:
Pushed!